connector-runtime/src/main/resources/application.properties

```diff
@@ -3,7 +3,6 @@ camunda.connector.polling.enabled=false
 camunda.connector.polling.interval=5000
 
 camunda.connector.webhook.enabled=false
-spring.main.web-application-type=none
```

Comment:
Wouldn't that always activate MVC REST endpoints?

Reply:
Restored - let's keep it disabled for now, I will test later if we really need this switch here since we have a few endpoints in the Runtime now.